examples/ip_adapter/README.md

```diff
@@ -0,0 +1,231 @@
+# IP-Adapter
+
+## Introduction
+
+This folder contains [IP-Adapter](https://arxiv.org/abs/2308.06721) models implemented with [MindSpore](https://www.mindspore.cn/), reference to [Official Implementation](https://github.com/tencent-ailab/IP-Adapter).
+
+IP-Adapter is an effective and lightweight adapter to achieve image prompt capability for the pre-trained text-to-image diffusion models. An IP-Adapter with only 22M parameters can achieve comparable or even better performance to a fine-tuned image prompt model. IP-Adapter can be generalized not only to other custom models fine-tuned from the same base model, but also to controllable generation using existing controllable tools. Moreover, the image prompt can also work well with the text prompt to accomplish multimodal image generation.
+
+<p align="center"><img width="700" alt="IP-Adapter Architecture" src="https://github.com/zhtmike/mindone/assets/8342575/f332b980-0fb9-4fe6-bc08-f66c33cabc8f"/>
+<br><em>Overall IP-Adapter architecture</em></p>
+
+## Dependency
+
+- mindspore 2.2
```

Comment:
Only ms 2.2 is supported? Does it mean running on 910A with ms2.1 is not feasible?

Reply:
em. as this model heavily relies on the original implemention of sdxl, as sdxl is only support 2.2 (see [here)](https://github.com/mindspore-lab/mindone/blob/master/examples/stable_diffusion_xl/GETTING_STARTED.md#dependency), so as this model

Reply:
I see. Thanks!